test/templates/egg-game.js

```diff
@@ -106,28 +106,33 @@ describe('loading authored state into template', () => {
           "drakes": [
             {
               "alleleString": nextState.drakes[0].alleleString, // we check valid alleles below
+              "secondXAlleles": null,
```

Comment:
In theory this could be skipped in all these tests -- and the state cleaned up a bit -- if `secondXAlleles` were only set if it were non-null in `load-state-from-authoring`. However, I think we have other `null` values scattered about the state, and we don't have a policy on removing them, so I think this is fine.